common/styles/components/_opening_hours.scss

```diff
@@ -80,42 +80,43 @@
 }
 
 .opening-hours__table {
+  max-width: 600px;
   color: color('silver');
   width: 100%;
   margin-bottom: 2 * $vertical-space-unit;
 
-  .opening-hours--light & {
+  &.opening-hours--light {
     color: color('black');
   }
 
-  .enhanced & {
+  .enhanced &.js-tabpanel {
```

Comment:
Do we have to use this `js-` class for styling? I think we've managed to avoid it so far.